View/encoding_table.py

```diff
@@ -173,6 +176,36 @@ def get_col_count(self):
         """
         return self.columnCount()
 
+    def get_cell_size(self):
```

Comment:
The implementation of the method looks like we're returning the minimum width and height of each cell. Should we rename the method accordingly? Up to you.